README.md

```diff
@@ -19,13 +19,17 @@ Step 1: install dependencies and run the ground station. In order to run as a us
 ```
```

Comment:
Gross title/description split. Probably just "use radioconda instead of manual dependency installation" is fine